refinery/ui/source/js/data-set-import/controllers/isa-tab-import.spec.js

```diff
@@ -0,0 +1,89 @@
+(function () {
+  'use strict';
+
+  describe('IsaTabImportCtrl', function () {
+    var ctrl;
+    var importConfirmationService;
+    var isaTabImportApiService;
+
+    beforeEach(function () {
+      module('refineryApp');
+      module('refineryDataSetImport');
+      inject(function ($controller, $rootScope, _isaTabImportApi_, _importConfirmationService_) {
+        isaTabImportApiService = _isaTabImportApi_;
```

Comment:
Previously we discussed avoiding the _syntax_. Dependencies should be alphabetical with outside libraries first and then alphabetical with our libraries. importConfir... before isaTab...